README.md

```diff
@@ -229,21 +215,7 @@ cd robot_ws
 Before you build the robot workspace, you must edit the file `robot_ws/src/object_tracker_robot/config/model_config.yaml` to include the location
 of your trained model.
 
-Create a file `/etc/ros/rosdep/custom-rules/object-tracker-rules.yaml` and add the following configuration to it.
-```bash
-libjpeg62:
-  ubuntu:
-    xenial: [libjpeg62]
-```
-
```

Comment:
Sure we don't need these anymore?

Reply:
I was able to `colcon bundle` without issues.

Reply:
colcon bundle won't necessarily detect missing dependencies in Python scripts (especially if they weren't declared in setup.py/package.xml but still used in the script).